cmd/api/auth/authz.go

```diff
@@ -7,6 +7,8 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/kubearchive/kubearchive/cmd/api/abort"
```

Comment:
How are all these changes in authz.go related to getting the kind from the resource name?

Reply:
See answer in the previous thread